docs/en/developers/06_Reporting_Bugs.md

```diff
@@ -0,0 +1,65 @@
+# Reporting a bug or a suggestion
+
+Despite the care given to FreshRSS, it's still possible that bugs occur. The project is young and development is dynamic, so it can be corrected quickly. You might also have a feature in mind that doesn't yet exist. Regardless whether your idea seems silly, far-fetched, useless or too specific, please don't hesitate to propose it to us! "Ideas in the air" often find an attentive ear. It's new external perspectives that make the project evolve the most.
+
+If you're convinced that you should be heard, here's how you can go about it.
+
+## On GitHub
+
+GitHub is the ideal platform to submit your requests. It allows us to discuss a problem or suggestion with others and it often generates new ideas. Let's not neglect this "social" aspect!
+
+ 1. [Go to the bug ticket manager](https://github.com/FreshRSS/FreshRSS/issues)
+ 2. Start by checking if a similar request hasn't already been made. If so, please feel free to add your voice to the request.
+ 3. If your request is new, [open a new bug ticket](https://github.com/FreshRSS/FreshRSS/issues/new)
+ 4. Finally, write your request. If you're fluent in English, it's the preferred language because it allows for discussion with the largest number of people.
+ 5. Please follow the tips below to make it easier to let your ticket be heard.
+ 
+## Informal
+
+Not everyone likes or uses GitHub for a variety of legitimate reasons. That is why you can also contact us in a more informal way.
+
+* On [our Mattermost chat](https://framateam.org/signup_user_complete/?id=e2680d3e3128b9fac8fdb3003b0024ee)
+* At events / meetings around Free Software
```

Comment:
There is also a subreddit https://www.reddit.com/r/freshrss/. Not very active but it works also.

Reply:
Ah, I did not know / had forgotten :-P